### ISSUE TITLE
Add term frontalis muscle overactivity

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10349]
**Preferred term label:**
Frontalis muscle overactivity

**Synonyms**
Overacting frontalis muscle, hyperactive frontalis muscle

**Definition (free text, please give PubMed ID)**
Frontalis muscle overactivity cause a characteristic staring and astonished facial expression
PMID: 16353177

**Parent term (use hpo.jax.org/app)**
Abnormality of frontalis muscle belly HP:3000004

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PSP-OMIM601104

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
added as HP:0430144
